xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/MyRecentModifications.xml

```diff
@@ -173,15 +173,33 @@
     <property>
       <content>{{velocity}}
 #set ($void = $services.async.useEntity("wiki:$xcontext.database"))
-#set ($query = $services.query.xwql('where doc.author = :author order by doc.date desc'))
+#set ($query = $services.query.xwql('select doc.fullName, doc.language from XWikiDocument doc where doc.author = :author order by doc.date desc'))
```

Comment:
You sure this works ? AFAIK you have you are supposed to use `Document` with xwql. Maybe it just go trough as is because the converter does not know it, but would be safer to use the standard from.

Reply:
Both are working yes.